src/main.js

```diff
@@ -1,12 +1,8 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
-import 'bootstrap/dist/css/bootstrap.css'
```

Comment:
Re-include bootstrap for now

Reply:
bootstrap modal styling in _modal.scss now